### ISSUE TITLE
Fully removing Fluent Forms and all associated data

### ISSUE THREAD

**Issue by paaljoachim** [fluentform/fluentform#455]
Hey

I was about to create a video tutorial showing how I would go about setting up the form.
I wanted to fully remove Fluent Forms as I wanted to start fresh showing what it looks like just after having installed and activated the form. But I am not able to find an option to fully remove Fluent Forms on plugin deletion.

As I expected to perhaps under Tools see a checkbox which I could select to fully remove Fluent Forms on plugin deletion.

How would I go about fully resetting Fluent Forms and remove everything on deleting the plugin?

**Comment by nkb-bd** [comment]
Try using database reset plugin

**Comment by paaljoachim** [comment]
Yes. I know that works, as I used Advanced Database cleaner plugin for this exact thing. 
But if one wants to fully remove Fluent Forms one should not need to add database reset plugin to remove all of the database entries that Fluent Forms does add. As it should be an option to remove the plugin fully. Some plugins do so and that is helpful for the user.

**Comment by paaljoachim** [comment]
Here is an example from Ninja Forms as it shows respect for the users web site:
![Screenshot 2024-07-25 at 09 50 17](https://github.com/user-attachments/assets/a6e3ed44-2a71-4210-8fea-0d864c52cfc9)

**Comment by justinclift** [comment]
Yeah, this would be useful in real world cases.  We've loaded dummy data into Fluent Forms while doing initial testing and learning with Fluent Forms (on our main Wordpress instance).  Now that we're looking to deploy it in a "proper" (production) sense, there's no buttuon to cleanly remove the data.

FluentCRM itself has one, so I was expecting the same from FluentForms.
